packages/cache-utils/src/fs.js

```diff
@@ -33,8 +34,9 @@ const isEmptyDir = async function ({ srcGlob, cwd, isDir }) {
     return false
   }
 
-  const files = await globby(srcGlob, { cwd, ignoreJunk: true })
-  return files.length === 0
+  const files = await globby(srcGlob, { cwd })
```

Comment:
One thing to note: `cpy` uses a few more options from `globby`:

https://github.com/sindresorhus/cpy/blob/e2be678e7908199d4aca6c2c5ba0a90b05078b19/glob-pattern.js#L50

However, those might require `globby` to be upgraded first since those are probably `fast-glob` options, not `glob`. We should also be careful to confirm the behavior did not change especially when it comes to:
  - Caching a directory as `dir` instead of `dir/**`
  - Dot files

Reply:
Agree. I'll look into those in the globby update PR 👍